utilities/write_batch_with_index/write_batch_with_index.cc

```diff
@@ -310,18 +322,21 @@ class BaseDeltaIterator : public Iterator {
           }
         } else {
           current_at_base_ = true;
-          return;
+          break;
         }
       }
     }
 
-    AssertInvariants();
+    IsOverUpperBound();
+    // AssertInvariants();
```

Comment:
Why remove assert?

Reply:
It seems to be a bug then. Now if you uncomment it, will it fail?

Reply:
yes, the list of tests that fail: env_test, optimistic_transaction_test, transaction_test, c_test, write_batch_with_index_test